primer-service/src/Primer/Server.hs

```diff
@@ -215,7 +216,7 @@ sessionsAPIServer :: ConvertServerLogs l => S.SessionsAPI (AsServerT (Primer l))
 sessionsAPIServer =
   S.SessionsAPI
     { S.createSession = newSession
-    , S.getSessionList = \p -> pagedDefault 100 p listSessions
+    , S.getSessionList = \s p -> pagedDefault 100 p $ maybe listSessions findSessions s
     , S.addSession = API.addSession
     , S.sessionAPI = sessionAPIServer
     }
```

Comment:
I'm confused here. The commit message says that in openapi we have one combined list+filter endpoint, but in "rich" api we have two separate ones. However, I thought this code was implementing the "rich" api, and it looks like we only have the combined one?

Reply:
Rich meaning Haskell, not either of the HTTP APIs.

Reply:
Oh, I see. Potentially worth rewording for clarity?